docs/self-hosted/kubernetes-maintenance.md

```diff
@@ -0,0 +1,27 @@
+# Maintenance
+## Private Packagist Self-Hosted (Kubernetes)
+
+### Updates
+
+By default Replicated checks for updates every 4 hours and you can install them from the dashboard once they are available.
```

Comment:
Replicated Management Console on port 8800

Shouldnt be "dashboard" only. This was a problem with the NREL user, he had absolutely no clue that this management console from KOTS exists and where to reach it. But he had the password (lol)